motoko/dao/tests/proposal.test.sh

```diff
@@ -0,0 +1,148 @@
+#!ic-repl
+load "prelude.sh";
```

Comment:
ic-repl looks nice - I should actually use it...

Reply:
But it also looks very sequential - are all calls made synchronous?

Reply:
All calls have an await, so the script is sequential. Do you have a use case to make concurrent calls?